### PR TITLE
Added Issue and PR Templates

### LINKED ISSUE #10: doc: added PR and Issue templates
## Description
We need PR and issue templates!

### PR DESCRIPTION
**Pull request by at-the-vr**
fixes #10 
This PR is to close Issue #10 , kindly let me know if it requires further changes

### CODE REVIEW

**Comment by vedant-jain03** [comment]
cc @yashikajotwani12

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+### Related issue
```
Comment (vedant-jain03) [inline .github/pull_request_template.md]: add prefix for closing the issue
add `fixes #<issue-number>` in the top!

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+<!-- Remove this field if it doesn't fix any issue -->
+
+### What changes does this PR do?
+
+### Screenshots (If any)
+
+### Questions
```
Comment (vedant-jain03) [inline .github/pull_request_template.md]: this is not needed, you can remove this!

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+### Related issue
+<!-- Remove this field if it doesn't fix any issue -->
+
```
Comment (vedant-jain03) [inline .github/pull_request_template.md:2]: Add 
```
Have you tested the working of the feature you fixed?
<-Yes or No->
```

File: .github/ISSUE_TEMPLATE/bug_issue.yml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+name: 🐛 Bug
```
Comment (vedant-jain03) [inline .github/ISSUE_TEMPLATE/bug_issue.yml:1]: It looks good but it should be simple enough I believe, required field should not be true!

Could you checkout this simple template, I like [this](https://github.com/CircuitVerse/CircuitVerse/blob/master/.github/ISSUE_TEMPLATE/bug_report.md) one.

Comment (vedant-jain03) [reply .github/ISSUE_TEMPLATE/bug_issue.yml:1]: Apply this in other templates as well!

**Review by vedant-jain03 — CHANGES_REQUESTED** [review]
@itsEobard2025 Thanks for raising the PR.

Here are my suggestions, check them out and let me know if you had any doubt!

File: .github/ISSUE_TEMPLATE/bug_issue.yml (hunk 4 above)
Comment (at-the-vr) [reply .github/ISSUE_TEMPLATE/bug_issue.yml:1]: Hey Vedant, thanks for the feedback, I have performed some changes in my [fork](https://github.com/itsEobard2025/certificate-generator). Could you test it out(making dummy issues/PRs) and tell me if it requires further enhancements ?

**Review by vedant-jain03 — APPROVED** [review]

**Pull request merged**